org.uqbar.project.wollok.lib/src/wollok/lib.wlk

```diff
@@ -346,8 +346,8 @@ object game {
 }
 
 class Position {
-	var property x
-	var property y
+	var x = 0
```

Comment:
This is because of TS problems?
I will put ts-properties on the top of my priorities.